src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -353,6 +354,15 @@ private EventDetails getLatestEvent() {
         return events != null && !events.isEmpty() ? events.get(0).getValue() : null;
     }
 
+    @JsonIgnore
+    public Hearing getLatestHearing() {
+        if (isNotEmpty(hearings)) {
+            getHearings().sort(Collections.reverseOrder());
+            return hearings.get(0);
```

Comment:
Why don't return hearings.get(hearings.size()-1); without sorting.

Reply:
Because the list might not be sorted before this, and if it were, it would still need to get the first one as it is sorted in reverse order. Does that make sense?